From: Gordon Ball <gordon@chronitis.net>
Date: Tue, 4 Feb 2020 08:30:54 +0000
Subject: Requirejs shim config for xterm, xterm-addon-fit

Due to lack of browserify, this module is concatenated (as of 3.8.1)
using browserify-lite. This produces modules with a global export
instead of a full UMD wrapper, so add a requirejs configuration for
this.
---
 tools/build-main.js | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/tools/build-main.js b/tools/build-main.js
index ba7bbdc..10a1b2c 100644
--- a/tools/build-main.js
+++ b/tools/build-main.js
@@ -55,6 +55,12 @@ var rjs_config = {
     "jquery-ui": {
       deps: ["jquery"],
     },
+    xterm: {
+      exports: "Terminal",
+    },
+    "xtermjs-fit": {
+      exports: "fit",
+    },
   },
 
   exclude: [
